.travis.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     - TEST_PROVISION_TIMEOUT=5m
     - TEST_BIND_TIMEOUT=5m
     - TEST_ASSERT_TIMEOUT=2m
+    # Forces the version.sh script to fetch tags and unshallow the git repository.
+    - VERSION_FORCE_TAG_FETCH=true
```

Comment:
Does making travis not do `--depth` help?
https://docs.travis-ci.com/user/customizing-the-build/#git-clone-depth

Reply:
It solves too, but most of the time we will not need to unshallow. E.g. as soon as we release 1.0.0-rc1, the shallow clone suffices.